contracts/gov/user-keeper/GovUserKeeper.sol

```diff
@@ -659,17 +692,25 @@ contract GovUserKeeper is IGovUserKeeper, OwnableUpgradeable, ERC721HolderUpgrad
         return _getFreeAssets(voter, false, lockedProposals, unlockedNfts);
     }
 
-    function getDelegatedAssetsAmount(
+    function getDelegatedStakeAmount(
         address delegator,
         address delegatee
     ) external view override returns (uint256) {
-        return _usersInfo[delegator].delegatedTokens[delegatee];
+        UserInfo storage userInfo = _usersInfo[delegator];
+
+        return
+            userInfo.delegatedTokens[delegatee] +
+            getTotalNftsPower(userInfo.delegatedNfts[delegatee].values());
```

Comment:
There is an optimization to pass only the number of delegated NFTs in case of non-power NFT

Reply:
done